src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
 spring.application.name=GitHubAiAPI
+
+
+Spring.demo.value= test
```

Comment:
It's a good practice to keep property names consistent. Consider renaming 'Spring.demo.value' to 'spring.demo.value' to maintain consistency.